xds/internal/xdsclient/xdsresource/unmarshal_lds_test.go

```diff
@@ -1852,17 +1852,18 @@ var customFilterNewTypedStructConfig = &v3xdsxdstypepb.TypedStruct{
 var wrappedCustomFilterNewTypedStructConfig *anypb.Any
 
 func init() {
-	wrappedCustomFilterOldTypedStructConfig = testutils.MarshalAny(customFilterOldTypedStructConfig)
-	wrappedCustomFilterNewTypedStructConfig = testutils.MarshalAny(customFilterNewTypedStructConfig)
+	t := &testing.T{}
```

Comment:
The whole idea of passing a `testing.T` to `testutils.MarshalAny` is to be able to fail that particular test when something errors out. This concept of passing an empty `testing.T` is not going to work.